### ISSUE TITLE
# symbol (as in C# or F#) is rendered as %23

### ISSUE THREAD

**Issue by mariosangiorgio** [airpair/airpair-com-v1-issue-tracker#96]
I'm completing my profile and I'm adding C# and F# as technologies I work with.

They are rendered fine in the popup that appears under the search box but they are rendered as C%23 or F%23 when they appear in the list where I can drag and reorder them
![screen shot 2015-01-24 at 01 24 12](https://cloud.githubusercontent.com/assets/153130/5885556/bb565282-a367-11e4-8c3e-be7bddece894.png)

**Comment by jkresner** [comment]
Yes this is a tricky one :)... I'll leave it in the backlog thank you.
